addon/templates/components/frost-autocomplete.hbs

```diff
@@ -0,0 +1,38 @@
+{{! Template for the frost-select component }}
+{{frost-text
+  class='frost-autocomplete-text'
+  ariaActiveDescendant=ariaActiveDescendant
+  ariaAutocomplete='list'
+  ariaExpanded=true
+  ariaMultiselectable=false
+  ariaOwns='frost-select-list'
+  hook=(concat hook '-text')
+  onInput=onFilterInput
+  onKeyDown=(action 'onKeyDown')
+  onKeyPress=(action 'onKeyPress')
+  onClear=(action 'onClear')
+  role='combobox'
+  value=filter
+  disabled=disabled
+  autofocus=_grabfocus
+  placeholder=placeholder
+}}
+
+{{#if _opened}}
+  {{to-elsewhere
+    named=renderTarget
+    send=(component 'frost-autocomplete-dropdown'
+      $element=$element
+      filter=filter
+      hook=(concat hook '-dropdown')
```

Comment:
Change this to `hook=(concat hook 'autocomplete-dropdown')`.  `frost-select` needs to have a similar change made as well so the sub-elements can have unique name scope even though similar elements and naming scheme.